assets/src/state.ts

```diff
@@ -826,34 +828,44 @@ const routeTabsPushInProgressReducer = (
 
 const routeTabsToPushReducer = (
   routeTabsToPush: RouteTab[] | null,
-  newRouteTabs: RouteTab[] | null,
+  routeTabsToPushNext: RouteTab[] | null,
+  newRouteTabs: RouteTab[],
   routeTabsUpdated: boolean,
   action: Action
-): RouteTab[] | null => {
+): [RouteTab[] | null, RouteTab[] | null] => {
```

Comment:
**suggestion:** Because the type doesn't make it clear, it's probably worth it to add a doc comment to this function to say that the order is `[routeTabsToPush, routeTabsToPushNext]`, so you don't have to scroll down to its call below to see that.